src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs

```diff
@@ -190,4 +247,27 @@ private static void ConfigureAttuContainer(EnvironmentCallbackContext context, M
         // This will need to be refactored once updated service discovery APIs are available
         context.EnvironmentVariables.Add("MILVUS_URL", $"{resource.PrimaryEndpoint.Scheme}://{resource.Name}:{resource.PrimaryEndpoint.TargetPort}");
     }
+
+    private static async Task<HealthCheckResult> HealthCheck(string? connectionString)
+    {
+        if (connectionString is null)
+        {
+            throw new InvalidOperationException("Connection string is unavailable");
+        }
+
+        try
+        {
+            var milvusClient = new MilvusClient(connectionString);
```

Comment:
Maybe the client should be created once? Do we want to have one client per call to health check? These run every 5 seconds...